flink-runtime-web/web-dashboard/app/partials/jobs/job.exceptions.jade

```diff
@@ -20,6 +20,10 @@
     .panel-title
       | Root exception
 
+  .panel-heading.clearfix
+    .panel-info.thin.last
+      | Timestamp: {{ exceptions['timestamp'] | amDateFormat:'YYYY-MM-DD, H:mm:ss' }}
```

Comment:
I would do it like this:
```
strong Timestamp:
= ' '
| {{ exceptions['timestamp'] | amDateFormat:'YYYY-MM-DD, H:mm:ss' }}
```

![screen shot 2017-03-21 at 11 20 02](https://cloud.githubusercontent.com/assets/1756620/24142965/777d370a-0e28-11e7-86e1-d4a61c6656cc.png)